Sprinkle of Yeezus/Settings.swift

```diff
@@ -11,52 +11,68 @@ import UIKit
 import UserNotifications
 
 class SettingsPage: UIViewController {
-    
 
-    @IBOutlet weak var InfoLabel: UILabel!
-    @IBOutlet weak var DescButton: UIButton!
-    @IBOutlet weak var DescLabel: UILabel!
-    @IBOutlet weak var Switch: UISwitch!
+    @IBOutlet private weak var InfoLabel: UILabel!
+    @IBOutlet private weak var DescButton: UIButton!
+    @IBOutlet private weak var DescLabel: UILabel!
+    @IBOutlet private weak var Switch: UISwitch!
+    @IBOutlet private weak var TimePicked: UIDatePicker!
     
-    var notificationsOn = Bool()
+    private var sprinkleTimePicked = Date()
+    private let notificationCenter = UNUserNotificationCenter.current()
     
-    @IBAction func NotificationSwitch(_ sender: UISwitch) {
-        if Switch.isOn == true{
-            UNUserNotificationCenter.current().requestAuthorization(options: [.alert, .sound], completionHandler: {didAllow, error in})
-            
-            DescLabel.isHidden = false
-            DescButton.isHidden = false
-            
-            notificationsOn = true
-        } else {
-            DescLabel.isHidden = true
-            DescButton.isHidden = true
-            notificationsOn = false
-        }
+    // MARK: - Lifecycle
```

Comment:
This mark is commonly referred to as a `pragma mark`, sort of adopted from the naming in Objective-C. Basically, this is a simple form of Documentation. What this does is separate the code below it from the code above it, and gives the section a name (in this case, the name is "Lifecycle" because that's what the `viewDidLoad` function is a part of)